ui/app/appointments/controllers/manage/list/appointmentsListViewController.js

```diff
@@ -74,6 +75,7 @@ angular.module('bahmni.appointments')
                     return appointmet;
                 });
                 $scope.searchedPatient = true;
+                $scope.tableInfo[2].enable = true;
```

Comment:
@amosh101 do we need this line when we are setting `$scope.searchedPatient = true;` above?

Reply:
Yes

Reply:
Please explain. Seems like we are duplicating logic.

Reply:
The enable field already points to $scope.searchedPatient, why are we overriding that?

Reply:
I believe we could actually delete the line above, let me test.

Reply:
nope, needs to go as is.

Reply:
ok then in that case using $scope.searchedPatient to set the enable field is not helpful for us since the value of tableInfo is not being updated when $scope.searchedPatient changes, which makes sense since there is no binding being established between those variables. For now please just update it to be enable=false

Reply:
Very well, updated it and will do a PR, but still gonna talk to Krishnanm to help me understand why removing line 77 even after this change affects the header.

Reply:
> Very well, updated it and will do a PR, but still gonna talk to Krishnanm to help me understand why removing line 77 even after this change affects the header.

Understood, it was not an issue.

Reply:
The $scope.searchedPatient is how we manage the display of the column content. Image below:
![Screenshot from 2019-05-22 12-15-54](https://user-images.githubusercontent.com/49514817/58167063-6ba84900-7c8b-11e9-81a2-c72bb12a8104.png)